src/orion/algo/hyperband.py

```diff
@@ -349,22 +403,24 @@ def suggest(self, num):
                 f"{self.__class__.__name__} cannot suggest new samples and must wait "
                 "for trials to complete."
             )
-
+        # TODO: Shouldn't this return None in this case?
         return []
```

Comment:
Either is fine. I think we should maybe eventually only return lists, which will be empty when the algo is unable to sample new trials. This would simplify a bit the handling of algo suggestions.

Reply:
Fixed in 3193cad11790c20076ca4de4b048aa00f6d76168 (not pushed yet)